util/repr-bytes/src/lib.rs

```diff
@@ -321,7 +321,7 @@ macro_rules! derive_serde_from_repr_bytes {
 
                     fn visit_seq<V>(self, mut seq: V) -> Result<Self::Value, V::Error>
                     where
-                        V: ::serde::de::SeqAccess<'de>,
+                        V: $crate::_exports::serde::de::SeqAccess<'de>,
```

Comment:
I encountered this when attempting to use this macro in a crate that imported serde under a different name.